datafusion/catalog-listing/Cargo.toml

```diff
@@ -28,6 +28,7 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
+apache-avro = { version = "0.17", optional = true }
```

Comment:
this is unfortunate -- is there any way we can avoid the (direct) avro dependency on the listing table? If not it is fine for this PR I was just wondering